java/examples/binary/README.md

```diff
@@ -0,0 +1,14 @@
+# s2i-java-binary-example
+
+This S2I example illustrates how to build a container image for a previously existing Java binary JAR. 
+In this example, one is copied from ../maven/target; but typically you instead would have it downloaded from somewhere.
+
+If you have a project with Java sources, you are probably more interested in using the ../maven/ or ../gradle/ examples, instead.
+
+## Usage
+
+    cp ../maven/target/*.jar deployments/
+    s2i build --copy . fabric8/s2i-java s2i-java-binary-example
+    docker run -p 8080:8080 s2i-java-binary-example
+
+See ../maven/README.md for more information.
```

Comment:
could be a real markdown link

Reply:
sure; done so in 35a09101185264eabe9d7ed58f376009f510ca64, also in line 6.